script/generate-regex.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs')
+const path = require('path')
+
+const regenerate = require('regenerate')
+
+const set = regenerate()
+  .add(require('unicode-12.1.0/Sequence_Property/Basic_Emoji/index.js'))
+  .add(require('unicode-12.1.0/Sequence_Property/Emoji_Flag_Sequence/index.js'))
+  .add(require('unicode-12.1.0/Sequence_Property/Emoji_Modifier_Sequence/index.js'))
+  .add(require('unicode-12.1.0/Sequence_Property/Emoji_Tag_Sequence/index.js'))
+  .add(require('unicode-12.1.0/Sequence_Property/Emoji_ZWJ_Sequence/index.js'))
+  .add(require('unicode-12.1.0/Block/Miscellaneous_Mathematical_Symbols_A/code-points.js'))
+  .add(require('unicode-12.1.0/Block/Miscellaneous_Mathematical_Symbols_B/code-points.js'))
+  .add(require('unicode-12.1.0/Block/Miscellaneous_Symbols/code-points.js'))
+  .add(require('unicode-12.1.0/Block/Miscellaneous_Symbols_And_Arrows/code-points.js'))
+  .add(require('unicode-12.1.0/Block/Miscellaneous_Symbols_And_Pictographs/code-points.js'))
+  .add(require('unicode-12.1.0/Block/Miscellaneous_Technical/code-points.js'))
```

Comment:
Alternatively the following also works, going by [General_Category](https://unicode.org/reports/tr44/#General_Category_Values) vs. Block/Sequence_Property:
```suggestion
  .add(require('unicode-12.1.0/General_Category/Close_Punctuation/code-points.js'))
  .add(require('unicode-12.1.0/General_Category/Open_Punctuation/code-points.js'))
  .add(require('unicode-12.1.0/General_Category/Other_Punctuation/code-points.js'))
  .add(require('unicode-12.1.0/General_Category/Symbol/code-points.js'))